tests/e2e/specs/psi-activation.test.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * WordPress dependencies
+ */
+import { visitAdminPage, activatePlugin, deactivatePlugin } from '@wordpress/e2e-test-utils';
+
+describe( 'PageSpeed Insights Activation', () => {
+	beforeEach( async() => {
+		await activatePlugin( 'e2e-tests-reset-plugin' );
+		await activatePlugin( 'e2e-tests-auth-plugin' );
+	} );
+
+	afterEach( async() => {
+		await deactivatePlugin( 'e2e-tests-auth-plugin' );
+		await deactivatePlugin( 'e2e-tests-reset-plugin' );
+	} );
```

Comment:
@swissspidy the reset plugin needs to be deactivated after activation or it will trigger a reset on every request.

Reply:
@aaemnnosttv Nope, it only does the reset in the plugin activation hook, not on every page load, as mentioned in my other comment.

Reply:
Yeah, I didn't see the change you made to the plugin itself 👍